recipes-extended/images/tb-minimal-image.bb

```diff
@@ -0,0 +1,15 @@
+# nooelint: oelint.file.requirenotfound
+require recipes-core/images/core-image-minimal.bb
+
+SUMMARY = "A small image just capable of allowing a device to boot."
+DESCRIPTION = "A small image just capable of allowing a device to boot."
+LICENSE = "MIT"
+
+IMAGE_FEATURES:append = " ssh-server-openssh"
+
+IMAGE_INSTALL:append = " \
+                        packagegroup-tb-base \
+                        packagegroup-tb-utils \
+                        "
+
+IMAGE_FSTYPES += "wic wic.gz wic.bmap"
```

Comment:
@m-iwanicki Here we are leacing both wic and wic.gz images. Running .wic image directly in qemu should be possible, with no need of manual unpacking afterwards.

Reply:
@macpijan Changed in https://github.com/3mdeb/meta-trenchboot/pull/30 to use `.wic` image.